CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Changelog
 
 ## [Unreleased]
+### Changed
```

Comment:
Are we planning to ship a release for `v17.0.0` once this is merged?
If so, let's add a header for `v17.0.0` along with a date in this PR as well.